entity-framework/core/modeling/owned-entities.md

```diff
@@ -31,6 +31,10 @@ If the `ShippingAddress` property is private in the `Order` type, you can use th
 
 [!code-csharp[OwnsOneString](../../../samples/core/Modeling/OwnedEntities/OwnedEntityContext.cs?name=OwnsOneString)]
 
+The model above is mapped to the following database schema:
+
+![image](_static/owned-entities-ownsone.png)
```

Comment:
You need to provide readable alt text describing the image instead of just "image"